charts/kubernetes/templates/topology.yaml

```diff
@@ -143,4 +143,40 @@ spec:
                   }).toJSON()
         properties:
           {{- include "kubernetes.topology.metricProperties.namespace" . | nindent 8 }}
+
+    - name: Ingresses
```

Comment:
Can we add a flag to enable or disable this ?

Reply:
👍🏽